backend/src/mirrors_qa_backend/routes/worker.py

```diff
@@ -0,0 +1,65 @@
+import pycountry
+from fastapi import APIRouter
+from fastapi import status as status_codes
+
+from mirrors_qa_backend.db.country import update_countries as update_db_countries
+from mirrors_qa_backend.db.worker import update_worker as update_db_worker
+from mirrors_qa_backend.routes.dependencies import CurrentWorker, DbSession
+from mirrors_qa_backend.routes.http_errors import BadRequestError, UnauthorizedError
+from mirrors_qa_backend.schemas import UpdateWorkerCountries, WorkerCountries
+from mirrors_qa_backend.serializer import serialize_country
+
+router = APIRouter(prefix="/workers", tags=["workers"])
+
+
+@router.get(
+    "/{worker_id}/countries",
+    status_code=status_codes.HTTP_200_OK,
+    responses={
+        status_codes.HTTP_200_OK: {
+            "description": "Return the list of countries the worker is assigned to."
+        }
+    },
+)
+def list_countries(worker_id: str, current_worker: CurrentWorker) -> WorkerCountries:
+    if current_worker.id != worker_id:
+        raise UnauthorizedError(
```

Comment:
Hum, this endpoint is for the public to check, not for the worker itself. What use would the worker make of this?